js/apps/admin-ui/src/components/dynamic/UrlComponent.tsx

```diff
@@ -0,0 +1,30 @@
+import { FormGroup } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+import { HelpItem } from "ui-shared";
+import { useFormContext, useWatch } from "react-hook-form";
+import type { ComponentProps } from "./components";
+import { FormattedLink } from "../external-link/FormattedLink";
+
+import "./url-component.css";
+
+export const UrlComponent = ({ name, label, helpText }: ComponentProps) => {
+  const { t } = useTranslation();
+  const { control } = useFormContext();
+  const { value } = useWatch({
+    control,
+    name: name!,
+    defaultValue: "",
+  });
+
+  return (
+    <FormGroup
+      label={t(label!)}
+      fieldId={name!}
+      labelIcon={<HelpItem helpText={t(helpText!)} fieldLabelId={`${label}`} />}
+      className="keycloak__identity-providers__url_component"
```

Comment:
Let's not add custom classes like this if we can avoid it.

Reply:
It's required to avoid misalignment of a non-editable component like this one. The actual CSS behavior has been inherited from https://github.com/keycloak/keycloak/blob/a7e5c861cc0623f39804b0b810c218d2febb0492/js/apps/admin-ui/src/identity-providers/add/saml-general-settings.css#L2